bot.js

```diff
@@ -62,39 +92,19 @@ const upvoteUrban = async (id) => {
         direction: "up"
       }
     });
-}
-
-const sendPatchNotes = async (send) => {
-  if (!send) {
-    return;
-  }
-  try {
-    const guilds = await client.guilds.fetch();
-    guilds.each(async (guild) => {
-      const guildRes = await guild.fetch();
-      let channels = await guildRes.channels.fetch();
-      channels.each((channel) => {
-        if (channel.type === 'GUILD_TEXT') {
-          if (channel.name === "general") {
-              channel.send('https://fuskdog.com');
-            }
-          }
-      });
-    });
   } catch (error) {
-    console.error(error);
-    return "error";
-  }
+    console.log(error);
+ }
 }
 
 const changeGuildNames = async (upvotes, isNewUpvote) => {
   try {
     const guilds = await client.guilds.fetch();
     guilds.each(async (guild) => {
       const guildRes = await guild.fetch();
-      // console.log(guildRes.name);
       if (isNewUpvote) {
         sendToChannels(guildRes, upvotes);
+        checkForGoals(guildRes, upvotes);
```

Comment:
You should combine checkForGoals with sendToChannels. Seems unnecessary to loop through all channels twice.